aw_datastore/storages/peewee.py

```diff
@@ -0,0 +1,113 @@
+from typing import Optional, List
+from datetime import datetime
+import json
+import os
+
+from peewee import Model, CharField, DateTimeField
+from playhouse.sqlite_ext import SqliteExtDatabase
+
+from aw_core.models import Event
+
+from . import logger, data_dir
+from .abstract import AbstractStorage
+
+# TODO: Make dependent on testing variable in constructor
+db = SqliteExtDatabase(os.path.join(data_dir, 'peewee-sqlite.db'))
+
+logger = logger.getChild("peewee")
+
+
+class BaseModel(Model):
+    class Meta:
+        database = db
+
+
+class BucketModel(BaseModel):
+    id = CharField(unique=True)
```

Comment:
I personally believe that we should replace all occurences of id, but it's not that important and i could live with it just being as it is, it's more of a nitpick. Also we're in early development so breaking the storage shouldn't matter much right now.